tests/test_autonomy/test_cli/test_analyse/test_abci/test_specs.py

```diff
@@ -229,6 +231,12 @@ def test_failures(
             "Please provide path to specification file." in result.output
         ), result.output
 
+    def teardown(
+        self,
+    ) -> None:
+        """Tear down the test."""
```

Comment:
you need to call `super().teardown()` o/w we don't have proper cleanup

Reply:
right